tests/video/conftest.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vim: ai ts=4 sts=4 et sw=4 nu
+
+import pytest
+import pathlib
+
+from zimscraperlib.video import VidCompressionCfg, VidUtil
+
+
+@pytest.fixture(scope="function")
+def vidcompressioncfg():
+    return VidCompressionCfg()
+
+
+@pytest.fixture(scope="function")
+def vidutil():
+    return VidUtil(VidCompressionCfg(), "mp4", True)
+
+
+@pytest.fixture(scope="function")
+def temp_video_dir():
```

Comment:
we want to use `tempfile` and not have that along the code. tests can fail so it may not be cleaned-up.